crates/next-error-code-swc-plugin/src/lib.rs

```diff
@@ -153,7 +153,9 @@ impl VisitMut for TransformVisitor {
         let error_message = error_message.unwrap();
 
         let code = self.errors.iter().find_map(|(key, value)| {
-            if *value == error_message {
+            // Normalize line endings by converting Windows CRLF (\r\n) to Unix LF (\n)
+            // This ensures the comparison works consistently across different operating systems
+            if value.replace("\r\n", "\n") == error_message.replace("\r\n", "\n") {
```

Comment:
Shouldn't the value already be normalized here? Otherwise a new error found on windows would be serialized with MS linefeeds

Reply:
> Shouldn't the value already be normalized here? Otherwise a new error found on windows would be serialized with MS linefeeds

You're right - let me just muate `error_message` directly

Reply:
Tested on Windows - no new error codes were generated now.